math/check_amicable_pair.cpp

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2020 @author iamnambiar
+ * 
+ * @file
+ * \brief A C++ Program to check whether a pair of number is [amicable pair](https://en.wikipedia.org/wiki/Amicable_numbers) or not.
+ * 
+ * \details
+ * Amicable Pair are two positive integers such that sum of the proper divisor 
+ * of each number is equal to the other number.
```

Comment:
```suggestion
 * of each number is equal to the other number.
 * @author iamnambiar
```
No copyrights. All code submitted will be submitted under the repo's MIT License. This is also mentioned in a check box when you create pull-requests.